hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/bucket/TestOMBucketCreateRequest.java

```diff
@@ -330,8 +332,10 @@ private void rejectBucketCreationHelper(String volumeName,
         String bucketName) {
     Throwable e = assertThrows(OMException.class, () ->
         doPreExecute(volumeName, bucketName));
-    assertEquals(e.getMessage(), "Invalid bucket name: " + bucketName);
-  }
+    assertEquals(
+        "INVALID_BUCKET_NAME bucket name has an unsupported character : _",
+        OMException.ResultCodes.INVALID_BUCKET_NAME + " " + e.getMessage()
+    );  }
```

Comment:
nit:
- no need to prepend `INVALID_BUCKET_NAME` to the messages
- unnecessary odd whitespace change

```suggestion
        "bucket name has an unsupported character : _",
        e.getMessage()
    );
  }
```